keyboards/handwired/numbrero/config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Flehrad
+#define PRODUCT         Numbrero
```

Comment:
lol I love your naming

Reply:
I didn't name it, Taylor/Drake named it :P